### ISSUE TITLE
No sources in spark-kotlin-1.0.0-alpha-sources.jar

### ISSUE THREAD

**Issue by StefanLiebig** [perwendel/spark-kotlin#24]
No sources in spark-kotlin-1.0.0-alpha-sources.jar in Maven

**Comment by perwendel** [comment]
No, as far as my knowledge stretches Kotlin source in releases to maven central isn't possible. I don't know if this has changed.

**Comment by StefanLiebig** [comment]
For configuration I use https://github.com/npryce/konfig and the Kotlin sources are available in maven: http://central.maven.org/maven2/com/natpryce/konfig/1.6.1.0/

**Comment by perwendel** [comment]
I'll see if I can get it to work
